client_updater.go

```diff
@@ -26,22 +26,21 @@ type ClientUpdate struct {
 // nopublishMessages is a set of message names that you don't send to clients, because they
 // contain no configuration that makes sense for clients to hear.
 var nopublishMessages = map[string]struct{}{
-	"CURRENTTIME":    {},
+	"CURRENTTIME": {},
 }
 
 // nologMessages is a set of message names that you don't log to the terminal, because they
 // are too long or too frequent to bother with.
 var nologMessages = map[string]struct{}{
-	"TRIGGERRATE":    {},
+	"TRIGGERRATE":     {},
 	"CHANNELNAMES":    {},
-	"ALIVE":    {},
```

Comment:
why is all this formatting changing? did gofmt rules change or something?

Reply:
I think my IDE (Atom) was supposed to be running go fmt and hasn't been for a long time. I tried running it at the command line, which yielded results on only some files. No idea why. Not happy about it, but not willing to track down the problem.